index.html

```diff
@@ -5,15 +5,95 @@
   <![endif]-->
 </head>
-
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
-</html>
\ No newline at end of file
+    
+
+  <!-- Copy and paste your HTML from the first UI project here -->
+  <div class="container">
+     <div class="title">
+       <div class="nav">
+        <a  href="#"> Services </a>
+        <a  href="#"> Product </a>
+        <a  href="#"> Vision </a>
+        <a  href="#"> Features </a>
+        <a  href="#"> About </a>
+        <a  href="#"> Contact </a>   
+      </div>    
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </div>
+ 
+   <div class="header">
+      <div class="header1">
+        <h1>
+         Innovation
+         <br> On
+         <br> Demand 
+        </h1>
+
+         <span><a href="#" class="get-started">Get Started</a></span>
+  
+       </div>
+       
+    <div class="header2">
+       <img src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </div>
+   
+   <hr>
+   <div class="content1">
+    <p> 
+       <span style="font-weight:bold">Features</span>
+      <br>
+      <br>
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+    <p> 
+     <span style="font-weight:bold">About</span>
+     <br>
+     <br>
+       Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+ </div>
+     <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  <div class="content2">
+     <p>
+         <span style="font-weight:bold">Services</span>
+      <br>
+         <br>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+   
+     <p><span style="font-weight:bold">Product</span>
+         <br>
+         <br>
+         Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+     <p> <span style="font-weight:bold">Vision</span>
+         <br>
+         <br>
+         Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+    </div>
+         <footer>
+     <hr><br>
+     <h4>Contact</h4>
+     <address>
+     <p>123 Way 456 Street<br>
+     Somewhere, USA</p>
+     <a href="1 (888) 888-8888">1 (888) 888-8888</a>
```

Comment:
In your contact section, because the address is in a `<p>` element, while the phone number and email address are in `<a>` elements, they don't line up due to a padding that you have set on your `<p>` elements in your mobile media query.